dashboard/app/models/sections/section.rb

```diff
@@ -606,6 +606,11 @@ def assigned_csa?
     script&.csa? || [CSA, CSA_PILOT_FACILITATOR].include?(unit_group&.family_name)
   end
 
+  def assigned_gen_ai?
+    ['exploring-gen-ai-1', 'exploring-gen-ai-2'].include?(script&.name) ||
```

Comment:
👋 Hi! So I gave these to Sam and didn't totally realize how they would be hard-coded into our system. Which means: I forgot to tell her the standalone unit names to use here too. And - I realized I made a mistake and forgot to include the year with the unit name. So taking both of these together, I think you'll need to update this array to be:

`['exploring-gen-ai1-2024', 'exploring-gen-ai2-2024', 'foundations-gen-ai-2024', 'customizing-llms-2024']`

Hope that makes sense - sorry for the tweak here

Reply:
For posterity, updated to this list!